lib/super_digit.rb

```diff
@@ -1,15 +1,16 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
Note that `n.digits` is an O(n) operation!

How do you think that impacts the time complexity.